jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -487,8 +490,16 @@ public void filter(ContainerRequestContext containerRequestContext) {
         Current.addActiveLRACache(lraId);
     }
 
-    private String createUriPrefix(ContainerRequestContext containerRequestContext) {
-        return ConfigProvider.getConfig().getOptionalValue("narayana.lra.base-uri", String.class)
+    private String createUriPrefix(ContainerRequestContext containerRequestContext, Class<?> resourceClass) {
+        return ConfigProvider.getConfig().getOptionalValue(LRAConstants.NARAYANA_LRA_BASE_URI_PROPERTY_NAME, String.class)
+                .map(s -> {
+                    String userUri = s.endsWith("/") ? s.substring(0, s.length() - 1) : s;
```

Comment:
Do you think there would be an expectation from a user point of view that the "/" might be put on the end again before returning out of the map function?

Reply:
This is for the LRA coordinator and it verifies the ServerLRAFilter already verifies this IIRC. So we always make correct callback URLs.